src/main/java/dev/amp/validator/ParsedHtmlTag.java

```diff
@@ -94,6 +96,164 @@ public String getValue(@Nonnull final String attrName, final int index) {
         return val;
     }
 
+    /**
+     * Gets the name attribute for an extension script tag.
+     *
+     * @return the name attribute
+     * @private
+     */
+    private String extensionScriptNameAttribute() {
+        if (this.upperName().equals("SCRIPT")) {
```

Comment:
Switch the check in case upperName() returns null.
if ("SCRIPT".equals(upperName())) {